packages/nextjs/app/page.tsx

```diff
@@ -1,10 +1,92 @@
 "use client";
 
+import { useEffect, useState } from "react";
 import Link from "next/link";
 import type { NextPage } from "next";
 import { BugAntIcon, MagnifyingGlassIcon } from "@heroicons/react/24/outline";
+import { useScaffoldReadContract } from "~~/hooks/scaffold-eth";
 
 const Home: NextPage = () => {
+  const [counter, setCounter] = useState<number | undefined>();
```

Comment:
We don't need a state here, we can use the counter directly in the render (see comment below).